<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendOnline</class>
 <widget class="QWidget" name="SendOnline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>562</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,1,0,1">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>391</width>
         <height>340</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>391</width>
         <height>340</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="sendButton">
        <property name="geometry">
         <rect>
          <x>207</x>
          <y>291</y>
          <width>181</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="cursor">
         <cursor>13</cursor>
        </property>
        <property name="toolTip">
         <string>Send MWC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="settingsButton">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>291</y>
          <width>181</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="toolTip">
         <string>Click to change settings. Number of change outputs and required confirmations.</string>
        </property>
        <property name="cursor">
         <cursor>13</cursor>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="contactsButton">
        <property name="geometry">
         <rect>
          <x>273</x>
          <y>63</y>
          <width>115</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Contacts</string>
        </property>
        <property name="toolTip">
         <string>Choose a contact to send to</string>
        </property>
        <property name="cursor">
         <cursor>13</cursor>
        </property>
       </widget>
       <widget class="QTextEdit" name="descriptionEdit">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>177</y>
          <width>379</width>
          <height>94</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Description</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="allAmountButton">
        <property name="geometry">
         <rect>
          <x>273</x>
          <y>117</y>
          <width>115</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>All</string>
        </property>
        <property name="toolTip">
         <string>Send all MWC in this account</string>
        </property>
        <property name="cursor">
         <cursor>13</cursor>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="sendEdit">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>63</y>
          <width>259</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Send to</string>
        </property>
       </widget>
       <widget class="QComboBox" name="accountComboBox">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>3</y>
          <width>382</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="amountEdit">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>117</y>
          <width>259</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Amount</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="control::MwcLabelProgress" name="progress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>43</width>
         <height>43</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>43</width>
         <height>43</height>
        </size>
       </property>
       <property name="text">
        <string>PROG</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelProgress</class>
   <extends>QLabel</extends>
   <header>control/MwcLabelProgress.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
